packages/front-end/components/Experiment/TabbedPage/ExperimentHeader.tsx

```diff
@@ -419,144 +622,131 @@ export default function ExperimentHeader({
-                  <button className="dropdown-item" onClick={duplicate}>
-                    Duplicate
-                  </button>
-                )}
-                {canRunExperiment && (
-                  <ConfirmButton
-                    modalHeader="Archive Experiment"
-                    confirmationText={
-                      <div>
-                        <p>Are you sure you want to archive this experiment?</p>
-                        {!safeToEdit ? (
-                          <div className="alert alert-danger">
-                            This will immediately stop all linked Feature Flags
-                            and Visual Changes from running
-                          </div>
-                        ) : null}
-                      </div>
+                  <DropdownMenuItem onClick={() => setAuditModal(true)}>
+                    Audit log
+                  </DropdownMenuItem>
+                </DropdownMenuGroup>
+                <DropdownMenuSeparator />
+                <DropdownMenuGroup>
+                  <DropdownSubMenu
+                    trigger={
+                      <Flex
+                        align="center"
+                        className={isWatching ? "font-weight-bold" : ""}
+                      >
+                        <PiEye style={{ marginRight: "5px" }} size={18} />
+                        <span className="pr-5">
+                          {isWatching ? "Watching" : "Not watching"}
+                        </span>
+                      </Flex>
                     }
-                    onClick={async () => {
-                      try {
-                        await apiCall(`/experiment/${experiment.id}/archive`, {
-                          method: "POST",
-                        });
-                        mutate();
-                      } catch (e) {
-                        console.error(e);
-                      }
-                    }}
-                    cta="Archive"
                   >
-                    <button className="dropdown-item" type="button">
-                      Archive
-                    </button>
-                  </ConfirmButton>
-                )}
-                {hasUpdatePermissions && experiment.archived && (
-                  <button
-                    className="dropdown-item"
-                    onClick={async (e) => {
-                      e.preventDefault();
-                      try {
-                        await apiCall(
-                          `/experiment/${experiment.id}/unarchive`,
-                          {
-                            method: "POST",
-                          }
-                        );
-                        mutate();
-                      } catch (e) {
-                        console.error(e);
-                      }
-                    }}
+                    <DropdownMenuItem
+                      onClick={async () => await handleWatchUpdates(true)}
```

Comment:
The DropdopMenuItem component's `onClick` prop doesn't support asychronous callbacks explicitly. This is working (and the error states work as well in that it doesn't cause an app crash). 

If anyone has a better pattern for this, let me know. We may just need to open a modal like we do everywhere else.